src/aws/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG MEDIAINFO_VERSION=20.09
 RUN wget https://mediaarea.net/download/binary/mediainfo/20.09/MediaInfo_CLI_${MEDIAINFO_VERSION}_Lambda.zip && \
     unzip MediaInfo_CLI_${MEDIAINFO_VERSION}_Lambda.zip -d /tmp/mediainfo
 
-FROM public.ecr.aws/lambda/nodejs:14 as core
+FROM amazon/aws-lambda-nodejs:14 as core
```

Comment:
s/repeared/repaired in the commit message.

I thought it was the contrary : rate limits on docker hub and aws registry down?

Reply:
No they temporarily published images based on `ARM64` instead `AMD64`. It's fixed now.